src/lib/flags.js

```diff
@@ -1,7 +1,7 @@
 import { version } from '../../package.json';
 
 // are we running bridge in a development build
-export const isDevelopment = import.meta.env.DEV === 'development';
+export const isDevelopment = import.meta.env.DEV === true;
```

Comment:
Should `isDevelopment` always be `true`? Even in production?

Reply:
@tomholford 
The logic above changes isDevelopment to be `true` if vite's env variable `meta.env.DEV` is set to the boolean `true`.

Previously it was checking if it was equal to the string `"development"` which would never be the case, because [that's not how that env variable works](https://vitejs.dev/guide/env-and-mode.html).

So before, `isDevelopment` was never true. Now, it will be true when in development mode. 

I'm not sure how exactly the site is deployed in production but vite should automatically be setting `meta.env.PROD` to `true` and `meta.env.DEV` to false on netlify/github/vercel. 

But yes, this would cause all local instances, including `preview:mainnet` to set `isDevelopment` to `true`. I think that's the intended logic here but let me know if it appears otherwise to you.

Reply:
Okay sounds good, thanks for the crosscheck.